appium/options/windows/windows/windows_options.py

```diff
@@ -0,0 +1,204 @@
+    APP_TOP_LEVEL_WINDOW = "appTopLevelWindow"
+    app_top_level_window = OptionsDescriptor[Optional[str], str](APP_TOP_LEVEL_WINDOW)
+    """
+    Set the hexadecimal handle of an existing application top level
+    window to attach to, for example 0x12345 (should be of string type).
+    Either this capability or app one must be provided on session startup.
+
+    Usage
+    -----
+    - Get
+        - `self.app_top_level_window`
+    - Set
+        - `self.app_top_level_window` = `value`
+    
+    Parameters
+    ----------
+    `value`: `str`
+
+    Returns
+    -------
+    - Get
+        - `Optional[str]`
+    - Set
+        - `None`
+    """
+
+
+class AppWorkingDirOption(SupportsCapabilities):
+    APP_WORKING_DIR = "appWorkingDir"
+    app_working_dir = OptionsDescriptor[Optional[str], str](APP_WORKING_DIR)
+    """
+    Set the full path to the folder, which is going to be set as the working
+    dir for the application under test. This is only applicable for classic apps.
+
+    Usage
+    -----
+    - Get
+        - `self.app_working_dir`
+    - Set
+        - `self.app_working_dir` = `value`
+    
+    Parameters
+    ----------
+    `value`: `str`
+
+    Returns
+    -------
+    - Get
+        - `Optional[str]`
+    - Set
+        - `None`
+    """
+
+
+class CreateSessionTimeoutOption(SupportsCapabilities):
+    CREATE_SESSION_TIMEOUT = "createSessionTimeout"
+    create_session_timeout = OptionsDescriptor[Optional[timedelta], Union[timedelta, int]](
+        CREATE_SESSION_TIMEOUT, tset=transform_duration_set
+    )
+    """
+    Set the timeout used to retry Appium Windows Driver session startup.
+    This capability could be used as a workaround for the long startup times
+    of UWP applications (aka Failed to locate opened application window
+    with appId: TestCompany.my_app4!App, and processId: 8480).
+
+    Usage
+    -----
+    - Get
+        - `self.create_session_timeout`
+    - Set
+        - `self.create_session_timeout` = `value`
+    
+    Parameters
+    ----------
+    `value`: `Union[timedelta, int]`
+
+    Returns
+    -------
+    - Get
+        - `Optional[timedelta]`
+    - Set
+        - `None`
+    """
+
+
+class ExperimentalWebDriverOption(SupportsCapabilities):
+    EXPERIMENTAL_WEB_DRIVER = "ms:experimental-webdriver"
+    experimental_webdriver = OptionsDescriptor[Optional[bool], bool](EXPERIMENTAL_WEB_DRIVER)
+    """
+    Enables experimental features and optimizations. See Appium Windows
+    Driver release notes for more details on this capability.
+
+    Usage
+    -----
+    - Get
+        - `self.experimental_webdriver`
+    - Set
+        - `self.experimental_webdriver` = `value`
+    
+    Parameters
+    ----------
+    `value`: `bool`
+
+    Returns
+    -------
+    - Get
+        - `Optional[bool]`
+    - Set
+        - `None`
+    """
+
+
+class WaitForAppLaunchOption(SupportsCapabilities):
+    WAIT_FOR_APP_LAUNCH = "ms:waitForAppLaunch"
+
+    @staticmethod
```

Comment:
I have already seen a duplicated transformer definition in WDA caps (e.g. allows to provide float seconds)

Reply:
since it is used in more than one capability moved the function to `transformers.py` 
`transform_duration_in_seconds_get` and `transform_duration_in_seconds_set`